src/runtime/cuda/cuda_device_api.cc

```diff
@@ -136,9 +136,10 @@ class CUDADeviceAPI final : public DeviceAPI {
                       size_t size,
                       DGLContext ctx_from,
                       DGLContext ctx_to,
-                      DGLType type_hint,
-                      DGLStreamHandle stream) final {
+                      DGLType type_hint) final {
+    auto stream = static_cast<DGLStreamHandle>(CUDAThreadEntry::ThreadLocal()->stream);
     cudaStream_t cu_stream = static_cast<cudaStream_t>(stream);
```

Comment:
Can be unified into one line:
```
cudaStream_t cu_stream = CUDAThreadEntry::ThreadLocal()->stream;
```

Reply:
lol.. Thanks. I was cpy/pasting without much thinking... Resolved as I will be outdated if I overload this function.